packages/did/src/commonHelpers.ts

```diff
@@ -0,0 +1,184 @@
+/**
```

Comment:
IMHO things like `commonHelpers`, `util` should be avoided by all means. It's not telling you what is contained at all and the separation to other modules is not given at all.

Reply:
> IMHO things like `commonHelpers`, `util` should be avoided by all means. It's not telling you what is contained at all and the separation to other modules is not given at all.

Yeah I hate it too. The only reason this exists is because anything else provokes problems with import cycles. Did not find another way to solve this issue, so our options are duplicating code or not checking service endpoints on the builder class at all. Note that for the end user this is invisible, because it is only exported via the utils file. And @tjwelde is working on reorganising the whole codebase, so this would be temporary in any case.

Reply:
We have been reorganising the code a little bit. Got rid of the commonHelpers and just put the function responsible for the circular dependency into a `Did.signature.ts` file